drivers/sensor/ite_vcmp_it8xxx2/Kconfig

```diff
@@ -0,0 +1,12 @@
+# ITE Voltage Comparator driver configuration options
+
+# Copyright (c) 2022 ITE Technology Corporation.
+# SPDX-License-Identifier: Apache-2.0
+
+config VCMP_IT8XXX2
+	bool "ITE it8xxx2 Voltage Comparator"
+	depends on SOC_IT8XXX2
```

Comment:
please default based on dt_compat_status.

Reply:
Do you mean dt_compat_enabled? I didn't find the usage of dt_compat_status in zephyr document, 
https://docs.zephyrproject.org/latest/guides/build/kconfig/preprocessor-functions.html?highlight=dt_compat

Reply:
This driver needs to depend on "ADC_ITE_IT8XXX2", or we can't get the adc device instance when call the adc_channel_setup().